MAUI/ImageEditor/imagefilter.md

```diff
@@ -162,15 +162,15 @@ public MainPage()
 
 {% endhighlight %}
 
-The applied effects can be canceled using the `CancelEdits` method or by calling the ImageEffect method with ImageEffect.None.
+The applied effects can be canceled using the [CancelEdits](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_CancelEdits) method or by calling the ImageEffect method with ImageEffect.None.

```

Comment:
or by calling the `ImageEffect`(method link) method with `ImageEffect.None`(link).